platform/features/plot/src/elements/PlotTelemetryFormatter.js

```diff
@@ -63,6 +65,10 @@ define(
         };
 
         PlotTelemetryFormatter.prototype.formatRangeValue = function (value) {
+            if (typeof value === 'number') {
```

Comment:
If value is a string representation of a number, this will delegate to the wrapped formatter and probably end up displaying the full string. 

Plots do have a policy restricting them to numeric telemetry, and none of our existing sources provide numeric telemetry as strings (although there are certainly valid reasons that one might do so in the future.) This case isn't handled in TelemetryFormatter on master, however ("1.23" will fail the `isNaN` check and, presumably, hit an exception on the `toFixed` call), so I'd consider it out of scope for the current issue.